enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/SdkSaveQuestionForm.tsx

```diff
@@ -6,14 +6,28 @@ import { SaveQuestionProvider } from "metabase/components/SaveQuestionForm/conte
 import { useCreateQuestion } from "metabase/query_builder/containers/use-create-question";
 import { useSaveQuestion } from "metabase/query_builder/containers/use-save-question";
 import { Stack, Title } from "metabase/ui";
+import type Question from "metabase-lib/v1/Question";
 
 import { useInteractiveQuestionContext } from "../context";
 
-export const SaveQuestion = ({ onClose }: { onClose: () => void }) => {
+export type SdkSaveQuestionFormProps = {
+  onSave?: (question?: Question) => Promise<void>;
```

Comment:
Should the `question` be non-nullable? As in `handleSave` we have a non-nullable question.

```suggestion
  onSave?: (question: Question) => Promise<void>;
```

Reply:
The user might not need the question in the `onSave` callback - they might want the data for some UI thing but they might not care